pkg/sources/github/github.go

```diff
@@ -1569,3 +1607,11 @@ func (s *Source) scanTarget(ctx context.Context, target sources.ChunkingTarget,
 
 	return common.CancellableWrite(ctx, chunksChan, chunk)
 }
+
+func removeURLAndSplit(url string) []string {
```

Comment:
I think this would result in a regression compared to `getRepoURLParts`. e.g., it doesn't work with `http://` urls.

Reply:
Ah, so the original refactor also fixed a bug? Is that fix something we can independently deploy?